src/main/java/uk/gov/ida/verifyserviceprovider/services/NonMatchingAssertionService.java

```diff
@@ -107,14 +116,6 @@ public void validateIdpAssertion( Assertion assertion,
         attributeStatementValidator.validate(assertion);
     }
 
-    private AssertionType classifyAssertion( Assertion assertion ) {
-        if (!assertion.getAuthnStatements().isEmpty()) {
-            return AssertionType.AUTHN_ASSERTION;
-        } else {
-            return AssertionType.MDS_ASSERTION;
-        }
-    }
-
     private TranslatedResponseBody translateAssertions( List<Assertion> assertions ) {
         return null;
     }
```

Comment:
We don't need this anymore.

Reply:
Yes the above logic, is implemented by a slight variation of your AssertionClassifier class only difference is that it has static members. Or maybe I don't get what you mean

Reply:
private TranslatedResponseBody translateAssertions( List<Assertion> assertions ) {
        return null;
    }

Reply:
Ok, i think i'll go with your recommendations, the private method was been used by the below, so for now I'll replace the call it it with return null;, instead of return translateAssertions(assertions)

  @Override
    public TranslatedResponseBody translateSuccessResponse( List<Assertion> assertions, String expectedInResponseTo, LevelOfAssurance expectedLevelOfAssurance, String entityId ) {
        validate(assertions, expectedInResponseTo, expectedLevelOfAssurance);
        return null;
    }

I'll also remove the interface **AssertionDataTranslator**

Reply:
Oh, I see, I hadn't realised anything was calling it.  I think something has gone quite wrong here.  translateSuccessResponse() should not be returning null.  Nothing is calling the translate() method that I wrote.  I think we took quite different approaches on the two branches, and they have not been integrated correctly.

Reply:
Ola's explained this to me; I was confused because I'd misunderstood the scope of the story.  We're not going to return a response yet, so it's fine that we're currently returning null.

For the same reason, it's not time to write any acceptance tests yet.